packages/components/src/date-time/stories/date.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import DatePicker from '../date';
+
+const meta: ComponentMeta< typeof DatePicker > = {
+	title: 'Components/DatePicker',
+	component: DatePicker,
+	argTypes: {
+		currentDate: { control: 'date' },
+		events: { control: { type: null } },
+	},
```

Comment:
Do you think there is any benefit to hiding the callback props from the controls table? Only the controls, not the generated docs table. According to the [Storybook docs](https://storybook.js.org/docs/react/essentials/controls#disable-controls-for-specific-properties) it should at least be possible.

Also, I get the impression the default story should pretty much be a playground for exploring the component. Can we enable editing the `events` array?

Reply:
> Do you think there is any benefit to hiding the callback props from the controls table? Only the controls, not the generated docs table. According to the [Storybook docs](https://storybook.js.org/docs/react/essentials/controls#disable-controls-for-specific-properties) it should at least be possible.

The controls for `onChange` and `onMonthPreviewed` are already hidden in the controls table because storybook sees that the type is `Function` and so doesn't have a control for that. Or did you mean something else?

> Also, I get the impression the default story should pretty much be a playground for exploring the component. Can we enable editing the events array?

Yeah good idea. Let's un-disable this so that folks can edit the JSON in the With Events story.

Reply:
> The controls for onChange and onMonthPreviewed are already hidden in the controls table because storybook sees that the type is Function and so doesn't have a control for that. Or did you mean something else?

I meant hiding it completely from the **_controls table_** not just hiding the control. The last part of the [linked docs section](https://storybook.js.org/docs/react/essentials/controls#disable-controls-for-specific-properties) details omitting something from the controls table but leaving it in the docs table. My thinking was just that if the user can't play with it, does it need to be in the controls table?

> Let's un-disable this so that folks can edit the JSON in the With Events story.

That aligns a little better with the TypeScript conversion guidelines that state:

> vii. Use the template for the Default story, which will serve as an interactive doc playground.